src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
+        for (Ball ball : balls) {
+            System.out.println(lottery.getRandomBall());
```

Comment:
You display not the same balls that you received before. And you get 3 more. Use from the balls array

Reply:
But it's best to remove the balls array altogether, just put a limit of 3 in the loop and create and output a new ball at the same time